apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SessionPoolManager.cs

```diff
@@ -307,6 +307,12 @@ private static async Task<SpannerClient> CreateClientAsync(SpannerClientCreation
                 new ChannelOption(GcpCallInvoker.ApiConfigChannelArg, apiConfig.ToString())
             };
 
+            if (!string.IsNullOrEmpty(channelOptions.AdditionalVersionHeaderName))
+            {
+                spannerSettings.VersionHeaderBuilder.AppendVersion(channelOptions.AdditionalVersionHeaderName,
```

Comment:
This will [fail](https://github.com/googleapis/gax-dotnet/blob/ab0c5a8895567066541c5567c113ac189288dd46/Google.Api.Gax/VersionHeaderBuilder.cs#L48) if more than one client is created with the same SessionPoolManager.